omnibus-arbitrum-sepolia.toml

```diff
@@ -0,0 +1,186 @@
+
+[setting.priceDeviationTolerance]
+defaultValue = "<%= parseEther('1') %>"
+
+[setting.spotStrictStalenessTolerance]
+defaultValue = "60"
+
+[setting.usdc_address]
+defaultValue = "0x75faf114eafb1BDbe2F0316DF893fd58CE46AA4d"
+description = "https://developers.circle.com/stablecoins/docs/usdc-on-test-networks"
+
+[setting.pythUsdcFeedId]
+defaultValue = "0xeaa020c61cc479712813461ce153894a96a6c00b21ed0cfc2798d1f9a9e9c94a"
+description = "https://pyth.network/developers/price-feed-ids#pyth-evm-mainnet"
+
+[setting.synthUsdcMaxMarketCollateral]
+defaultValue = "<%= parseEther(String(500_000)) %>"
+
+[setting.synthUsdcAsyncFixedFee]
+defaultValue = "<%= parseEther('0.1') %>" # FIXME
+
+[setting.synthUsdcSkewScale]
+defaultValue = "<%= parseEther(String(100_000)) %>" # FIXME
+
+[setting.dai_address]
+defaultValue = "0x"
+description = "Using mock for testnet"
+
+[setting.pythDaiFeedId]
+defaultValue = "0xb0948a5e5313200c632b51bb5ca32f6de0d36e9950a942d19751e833f70dabfd"
+description = "https://pyth.network/developers/price-feed-ids#pyth-evm-mainnet"
+
+[setting.synthDaiMaxMarketCollateral]
+defaultValue = "<%= parseEther(String(500_000)) %>"
+
+[setting.synthDaiAsyncFixedFee]
+defaultValue = "<%= parseEther('0.1') %>" # FIXME
+
+[setting.synthDaiSkewScale]
+defaultValue = "<%= parseEther(String(100_000)) %>" # FIXME
+
+[setting.weth_address]
+defaultValue = "0x980B62Da83eFf3D4576C647993b0c1D7faf17c73"
+description = "https://docs.arbitrum.io/build-decentralized-apps/reference/useful-addresses#core-contracts-1"
+
+[setting.pythEthFeedId]
+defaultValue = "0xff61491a931112ddf1bd8147cd1b641375f79f5825126d665480874634fd0ace"
+description = "https://pyth.network/developers/price-feed-ids#pyth-evm-mainnet"
+
+[setting.arb_address]
+defaultValue = "0x"
+description = "Using mock for testnet"
+
+[setting.pythArbFeedId]
+defaultValue = "0x3fa4252848f9f0a1480be62745a4629d9eb1322aebab8a791e344b3b9c1adcf5"
+description = "https://pyth.network/developers/price-feed-ids#pyth-evm-mainnet"
+
+[provision.dai_mock_collateral]
+source = "mintable-token:1.8"
+options.name = "Fake DAI Coin"
+options.symbol = "fDAI"
+options.owner = "<%= settings.owner %>"
+options.decimals = "18"
+
+[provision.arb_mock_collateral]
+source = "mintable-token:1.8"
+options.name = "Fake ARB Coin"
+options.symbol = "fARB"
+options.owner = "<%= settings.owner %>"
+options.decimals = "18"
+
+[invoke.setScPoolConfig]
+target = ["system.CoreProxy"]
+fromCall.func = "getPoolOwner"
+fromCall.args = ["<%= settings.sc_pool_id %>"]
+func = "setPoolConfiguration"
+args = [
+    "<%= settings.sc_pool_id %>",
+    [
+        { marketId = "<%= extras.synth_dai_market_id %>", weightD18 = 1, maxDebtShareValueD18 = "<%= parseEther('1') %>" },
+        { marketId = "<%= extras.synth_usdc_market_id %>", weightD18 = 1, maxDebtShareValueD18 = "<%= parseEther('1') %>" },
+        { marketId = "<%= imports.perpsFactory.settings.superMarketId %>", weightD18 = 1, maxDebtShareValueD18 = "<%= parseEther('1') %>" },
```

Comment:
Probably want significantly more weight for perps market over the spot markets/wrappers

Reply:
What value is needed? 10? 100?

Reply:
I'm not entirely sure. The question is basically what proportion of the collateral should be dedicated to backing sUSDC and sDAI that doesn't have corresponding wrapped assets. Presumably little of this would exist. Maybe kick up line 172 to `98` for now, so its 1% for DAI and 1% for USDC, and we can have some others weigh in.

Reply:
There is a code suggestion in github comment. Very useful
![image](https://github.com/Synthetixio/synthetix-deployments/assets/28145325/79f33d8d-9087-4d2c-9053-d558c86c222a)

```suggestion
        { marketId = "<%= imports.perpsFactory.settings.superMarketId %>", weightD18 = 98, maxDebtShareValueD18 = "<%= parseEther('1') %>" },
```